### ISSUE TITLE
Null Argument Internal Error when right clicking table column header

### ISSUE THREAD

**Issue by mrcbrown1** [portfolio-performance/portfolio#4309]
The following dialog is presented when right clicking on the column header bar of the Calculation tab of the Performance Calculation pane in version 0.71.2 of the software.
![image](https://github.com/user-attachments/assets/fed3cb14-573c-4303-a573-31542f8778e4)

The red X marks the spot
![image](https://github.com/user-attachments/assets/ed5ebaea-e5a9-46ed-8a0e-37991788351c)

**To Reproduce**
1. Go to the 'Performance Calculation' pane
2. Select the Calculation Tab
3. Right click in the region shown by the red X
4. All going (well) wrong, you see the error.

**Desktop (please complete the following information):**

 - OS: Windows 11
 - Version 0.71.2 (October 2024)
 - Platform win32, x86_64
 - Java: 17.0.5+8-LTS

**Comment by Nirus2000** [comment]
Hello @mrcbrown1 
I tried to reproduce your error, but it did not work.
Can you send us the error log? (Menu --> Help --> Save error log)

You can also use https://screenpal.com/ to record the individual steps on video.

Thank you very much
Alex

**Comment by mrcbrown1** [comment]
Hey Alex,

Absolutely - error log attached here:
[pp-error-2024-10-26.log](https://github.com/user-attachments/files/17529424/pp-error-2024-10-26.log)

I stripped out errors relating to API keys and kept only the session from this morning.

I've taken a video showing the error, and also how it doesn't occur on all table elements, it's here:

https://youtu.be/Ibxg36CjhP4

I've got mouse coordinate highlighting on and when I left/right click there are yellow/blue circles to show that event.

Let me know if I can provide any more info!

Cheers,
Mitch
